vms/platformvm/camino_vm_test.go

```diff
@@ -61,11 +63,22 @@ func TestRemoveDeferredValidator(t *testing.T) {
 	defer vm.ctx.Lock.Unlock()
 
 	// Set consortium member
-	// add admin proposer role to root admin
+	// set admin proposer role
```

Comment:
```suggestion
	// set ConsortiumSecretary role
```
I think we should be consistent with the naming. It's somewhat confusing.